x-pack/plugins/reporting/public/share_context_menu/index.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import * as Rx from 'rxjs';
```

Comment:
was this moved from somewhere ?

Reply:
These types were duplicated in `share_context_menu/register_csv_reporting.tsx` and `share_context_menu/register_pdf_png_reporting.tsx`